packages/suite/src/actions/wallet/formDraftActions.ts

```diff
@@ -0,0 +1,55 @@
+import { GetState } from '@suite-types';
+import { getFormDraftKey } from '@wallet-utils/formDraftUtils';
+import { FormDraftKeyPrefix, FormDraft } from '@wallet-types/formDraft';
+import { Dispatch } from 'redux';
```

Comment:
```suggestion
import type { Dispatch, GetState } from '@suite-types';
```
use internal `Dispatch` type

Reply:
Fixed.